pkgs/os-specific/linux/bpftrace/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchFromGitHub
+, cmake, pkgconfig, flex, bison
+, llvmPackages, kernel, linuxHeaders, elfutils, libelf, bcc
+}:
+
+stdenv.mkDerivation rec {
+  name = "bpftrace-${version}";
+  version = "git-2018-10-16";
```

Comment:
Should rather be:
```nix
  name = "bpftrace-unstable-${version}";
  version = "2018-10-16";
```

to follow [package naming conventions](https://nixos.org/nixpkgs/manual/#sec-package-naming)

Reply:
Thanks, fixed.